src/core/cp-calculator/cp-calculator.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { expect } from 'chai';
+import { CPCalculator } from './cp-calculator';
+
+describe('Class: CPCalculator', () => {
+    it('should output the correct values', () => {
+        // Source: http://www.imore.com/pokemon-go-gen-2
+        let cp: number;
+        // Bayleef
+        cp = CPCalculator.Calculate(122, 155, 120);
```

Comment:
This looks like the wrong argument order to me. I got the right answer with `cp = CPCalculator.Calculate(120, 122, 155);`.